src/main/common/router/routerFinder.ts

```diff
@@ -1,8 +1,11 @@
+import * as path from 'path'
 import { Router } from 'express'
 import * as requireDirectory from 'require-directory'
 
+const fileExtension: string = path.extname(__filename).slice(1)
```

Comment:
If `ts-node` process runs an application the extension is equal `ts`. If node is used then extension is obviously `js`.